com.unity.multiplayer.mlapi/Runtime/Core/MessageBatcher.cs

```diff
@@ -195,6 +199,7 @@ public int ReceiveItems(in BitStream messageStream, ReceiveCallbackType receiveC
 
                 if (rpcSize < 0)
                 {
+                    copy.Dispose();
```

Comment:
If we are using "using" do we still need the "Dispose()" ?

Reply:
nope, using does `try { ... } finally { obj.Dispose(); }` for us

Reply:
I think we should not use using that way.
It must have not been taken out, I believe that usage is C# 8.0 which last I checked we were not going to support at this time.
The real fix here is to replace this:
`using PooledBitStream copy = PooledBitStream.Get();`
with this:
`PooledBitStream copy = PooledBitStream.Get();`